test/goldens/end2end/pub_server-0.1.4-2.json

```diff
@@ -904,6 +904,12 @@
         "grantedPoints": 20,
         "maxPoints": 20,
         "summary": "Package gets 20 points if its Dart sdk constraint allows Dart 2."
+      },
+      {
+        "title": "Packaging conventions",
+        "grantedPoints": 15,
+        "maxPoints": 20,
+        "summary": "Package gets 10 points for a valid pubspec.yaml and 5 points for each of README.md and CHANGELOG\n\nNo CHANGELOG found.\nChangelog entries help developers follow the progress of your package. See the [example](https://raw.githubusercontent.com/dart-lang/stagehand/master/templates/package-simple/CHANGELOG.md) generated by `stagehand`."
```

Comment:
Add a `.` before the `\n\n`? (This affects tests.)

Reply:
Done!